### ISSUE TITLE
Eyes foreign body sensation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9806]
**Preferred term label:**
Eyes foreign body sensation

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Blepharitis, infectious

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
is this sufficiently covered by

Corneal foreign body sensation
HP:0034804
(new term, not yet online)
?

**Comment by MickeySegal** [comment]
Perfect.

**Comment by pnrobinson** [comment]
super, closing
